ert-runner.el

```diff
@@ -150,9 +150,9 @@ primarily intended for reporters."
   "Run tests matching PATTERN."
   (ert-runner/add-selector pattern))
 
-(defun ert-runner/load (&rest load-files)
+(defun ert-runner/load (load-files)
```

Comment:
The variable name should be `load-file` and that variable should be appended to the `ert-runner-load-files` list.